internal/cluster/endpoints/common.go

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoints
+
+import (
+	"github.com/banzaicloud/pipeline/config"
+	"github.com/sirupsen/logrus"
+)
+
+// nolint: gochecknoglobals
+var log logrus.FieldLogger
```

Comment:
New code **MUST** not use a global logger. Please inject one (see new modules, like issue, notification, clusterfeature for examples)

Reply:
fixed, thx @siliconbrain